tic-tac-toe/src/app/screens/Login/index.js

```diff
@@ -6,30 +6,42 @@ import loginActions from '@redux/Login/actions';
 import Login from './layout';
 
 class LoginContainer extends Component {
+  state = {
+    show: true
+  };
+
   onSubmit = values => {
+    this.setState({ show: true });
     const { username, password } = values;
     this.props.dispatch(loginActions.login(username, password));
   };
 
+  closeModal = () => {
+    this.setState({ show: false });
```

Comment:
i think you can extract this setState and use prevState to set true or false. So you can use that to close or in the submit function